README.md

```diff
@@ -122,6 +122,24 @@ For NetworkManager, a connection can only be active at one device each time.
 You can also use the same connection profile multiple times. Therefore, it is possible
 to create a profile and activate it separately.
 
+**Note:** The network role will only change the profiles that are specified in the
+`network_connections` variable. Therefore, if only the ports of a profile are specified
+to be removed from the controller and the controller is not specified, then the
+controller profile will remain on the system. This can happen, if for example all ports
+are removed from a bond interface.
+
+**Note:** To remove all profiles on a system that are not specified in the
+`network_connections` variable, add an entry without a name and `persistent_state:
+absent`. This will match and remove all remaining profiles:
```

Comment:
Does inserting a newline between different connections look better ? 

```yaml
network_connections:
  - name: eth0  # profiles to keep/configure on the system
    [...]

  - persistent_state: absent  # remove all other profiles
```

Reply:
Thanks, good catch, fixed it.